packages/neuron-wallet/src/block-sync-renderer/sync/indexer-cache-service.ts

```diff
@@ -245,6 +238,44 @@ export default class IndexerCacheService {
     return newTxHashes
   }
 
+  public static async upsertIndexerCache(
+    txs: {
+      txHash: string
+      txIndex: string
+      blockNumber: string
+      lockHash: string
+      address: string
```

Comment:
For inserting into the `indexer_tx_hash_cache` table, I generate the `lockHash` and `address` in advance based on the `script` at https://github.com/yanguoyu/neuron/blob/fix-light-tx-type/packages/neuron-wallet/src/block-sync-renderer/sync/light-connector.ts#L145-L150

Reply:
We may optimize fields of https://github.com/yanguoyu/neuron/blob/fix-light-tx-type/packages/neuron-wallet/src/database/chain/entities/indexer-tx-hash-cache.ts later because `lock hash` should be enough.